scripts/publishToConfluence.gradle

```diff
@@ -68,9 +70,8 @@ task publishToConfluence(
         config.confluence.useV1Api = findProperty("confluence.useV1Api") != null ?
             findProperty("confluence.useV1Api") : config.confluence.useV1Api != [:] ?
             config.confluence.useV1Api :true
-        binding.setProperty('config',config)
-        binding.setProperty('docDir',docDir)
-        evaluate(new File(projectDir, 'core/src/main/groovy/org/docToolchain/scripts/asciidoc2confluence.groovy'))
+        println("Using dev version!")
```

Comment:
Please delete this single debug print ✌️

Reply:
Done. :+1: